app/src/main/java/com/android/shelfLife/model/recipe/Recipe.kt

```diff
@@ -1,19 +1,45 @@
 package com.android.shelfLife.model.recipe
 
-import com.android.shelfLife.model.foodFacts.FoodFacts
+import com.android.shelfLife.model.foodFacts.NutritionFacts
+import com.android.shelfLife.model.foodFacts.Quantity
+import com.android.shelfLife.model.foodItem.FoodItem
 import kotlin.time.Duration
 
+/** Data class representing a recipe object */
 data class Recipe(
+    val uid: String, // unique identifier for the recipe
     val name: String, // recipe name
     val instructions: List<String>, // instructions of recipes step by step (hence the list)
-    val servings: Int, // total number of servings
+    val servings: Float, // total number of servings
     val time: Duration, // time it takes to cook
-    val ingredients: List<Ingredient> =
-        listOf(), // ingredients in recipe todo check with Alex about this
-    val recipeTypes: List<RecipesRepository.SearchRecipeType>
+    val ingredients: List<Ingredient> = listOf(), // ingredients in recipe
```

Comment:
As a future feature maybe we should also add other fields describing the recipe such as what ethnicity it is eg mexican, korean, italian or if its a breakfast, lunch or diner recipe